.test-defs/DeleteManagedSeed.yaml

```diff
@@ -0,0 +1,16 @@
+kind: TestDefinition
+metadata:
+  name: delete-managed-seed
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Tests the deletion of a managed seed.
+  activeDeadlineSeconds: 3600
```

Comment:
the graceful test delete timeout is also 1 hours so the testmachinery could kill the test before the test runs into its own timeout.
Lets increase it by 10 minutes to be save.

Reply:
I am not sure I understand this. If this is the case, shouldn't we actually decrease the timeout by 10 minutes rather than increasing it? Also, is this only applicable to the `DeleteManagedSeed` definition but not to the `CreateManagedSeed`?

Reply:
so currently there are 2 timeouts, one that is used by the testmachinery and one that is used by the go test.

When the go test timeout is reached, the test get into fail state and the cleanup handlers + dump of the state executed.
When the testmachinery timeout is reached the pod, that runs the go test, will get deleted. This means that if the testmachinery is reached before the go test timout no celanup handlers or dumps are executed.

So it is better to have the testmachinery timeout only as safety mechanism if the test timout does not work.

> Also, is this only applicable to the DeleteManagedSeed definition but not to the CreateManagedSeed?
Its also applicable in the create managed seed definition

Reply:
OK, I think I got it now. Increased this timeout to 4200 as suggested, the `CreateManagedSeed` timeout was already 7200 so no need to touch it I guess.